pentaho-js/src/main/javascript/cdf-legacy/components/Pentaho.Analyzer.js

```diff
@@ -109,29 +109,24 @@ var ExecuteAnalyzerComponent = AnalyzerComponent.extend({
             }
         });
     },
+
     executeAnalyzerComponent: function() {
         var callVar = this.isEditMode() ? "editor" : "viewer";
         var parameters = this.getOptions();
-        var path = {};
-        if ( parameters.solution ) {
-            $.extend( path, {solution: parameters.solution});
-        }
-        if ( parameters.path ) {
-            $.extend( path, {path: parameters.path});
-        }
-        if ( parameters.action ) {
-            $.extend( path, {action: parameters.action});
-        }
-        delete parameters.solution;
-        delete parameters.path;
-        delete parameters.action;
-        $.extend( parameters, {ts: new Date().getTime()});
+
+        parameters.ts = new Date().getTime();
```

Comment:
Are you sure that the following removed lines are not required?
```js
       delete parameters.solution;
        delete parameters.path;
        delete parameters.action;
```

Reply:
Yes, they were supposed to be removing this values from the object, to avoid passing them as query params to the API request.

But this values were populated using the component's (solution, path and action) properties in the `getOptions` method. (In other components, because in ExecuteAnalyzerComponent we were ignoring their values).

Now they are never there, so we don't need to delete those values